recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.0.2" %}
+
+package:
+  name: nibabel
+  version: {{ version }}
+
+source:
+  fn: nibabel-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/n/nibabel/nibabel-{{ version }}.tar.gz
+  md5: fa89163149e35f52e237b2cb56b1c606
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+      features:
+          - vc9     [win and py27]
+          - vc10     [win and py34]
+          - vc14      [win and py35]
```

Comment:
Move these from `requirements/build` to the `build` section above. Also, add `#` in front of each selector with a space between the two. Would be nice if they were indented to match.